tests/commons/j2/test_j2_functions.py

```
@@ -272,6 +276,101 @@
+    ],
+)
+def test_auto_date(case: tuple[str, str, str | None]) -> None:
+    assertCase("auto-date", case, [])  # no prefix-variants to test
+
+
+def test_fail_auto_date() -> None:
+    with pytest.raises(ValueError):
+        xsd_fmt("brol", "auto-date")
+        raise Exception(
+            "auto-date should not accept strings that have no meaningfull date format"
+        )
```

Ruff (0.8.2)

`pytest.raises()` block should contain a single simple statement

(PT012)